src/System.Runtime/tests/System/Reflection/FieldInfoTests.cs

```diff
@@ -9,7 +9,11 @@
 
 namespace System.Reflection.Tests
 {
+#if MONO
```

Comment:
Why do we need to make it partial?

Reply:
To avoid a conflict with similar named class from System.Reflection

Reply:
../../../external/corefx/src/System.Reflection/tests/FieldInfoTests.cs
../../../external/corefx/src/System.Runtime/tests/System/Reflection/FieldInfoTests.cs

Reply:
That seems to me as a problem on CoreFX side. All FieldInfoTests tests should be in System.Reflection. Please fix this upstream.